SingularityService/src/main/java/com/hubspot/singularity/config/SingularityConfiguration.java

```diff
@@ -81,6 +81,8 @@
 
   private boolean defaultValueForKillTasksOfPausedRequests = true;
 
+  private int defaultDeployStepWaitTimeSeconds = 0;
```

Comment:
I've learned my lesson with seconds. Always make millis so you can test it easier and be consistent.